src/pathwayvis/components/map/map.component.ts

```diff
@@ -295,6 +295,7 @@ class MapComponentCtrl {
     } else if (type === ObjectType.Reference) {
       if (!settings.model_id) return;
       const addedReactions = this._mapOptions.getAddedReactions();
+      const infoPromise = this._api.getWildTypeInfo(settings.model_id);
```

Comment:
I'm not really happy with the variable name (in general `data` and `info` sounds a bit vague IMO). Could you please just inline it? I think it's not being used anywhere else.